README.md

```diff
@@ -66,121 +64,20 @@ virtual machine to build images, make sure to allocate sufficient disk
 space.
 
     The resulting ipk files that can be installed through opkg exist at
-the following directory:
-
-        tmp-glibc/deploy/ipk/...
-
-4. (Optional/Advanced) Bitbake can transform tmp-glibc/deploy/ipk/ into
```

Comment:
We could still retain this pointer. Could be useful.

Reply:
I deleted it because it's really only a partial workflow. Creating a feed index doesn't really get you anywhere unless you understand how to add the feed to your opkg configuration and either disable feed signing, trust the feed explicitly, or install your own signing key. Again, it would be nice to have that in a tutorial or a wiki somewhere, but putting it all in the README seems excessive.